src/generate.js

```diff
@@ -0,0 +1,54 @@
+const fs = require('fs');
```

Comment:
this is super cool. I am thinking the non-generation option of option 2 is cleaner. What do you think?